tycho-p2/tycho-p2-director-plugin/src/main/java/org/eclipse/tycho/plugins/p2/director/DirectorMojo.java

```diff
@@ -55,6 +55,21 @@
     /** @parameter default-value="true" */
     private boolean installFeatures;
 
+
+    /**
+      * Additional install units that may be required to complete the product installation	  
```

Comment:
Not necessary required. So it could just say `Additional installable units to include with the product`.
Is it possible to enforce a specific version and if yes, what's the syntax to be used? Documentation should clarify that.